modules/virt-runbook-virtoperatorresterrorsburst.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-runbooks.adoc
+
+:_content-type: REFERENCE
+[id="virt-runbook-virtoperatorresterrorsburst_{context}"]
+= VirtOperatorRESTErrorsBurst
+
+// Edited by Jiří Herrmann, 8 Nov 2022
+
+[discrete]
+[id="meaning-virtoperatorresterrorsburst_{context}"]
+== Meaning
+
+This alert fires when more than 80% of the REST calls in the `virt-operator`
+pods failed in the last 5 minutes. This usually indicates that the `virt-operator`
+pods cannot connect to the API server.
+
+This error is frequently caused by one of the following problems:
+
+* The API server is overloaded, which causes timeouts. To verify if this is
+the case, check the metrics of the API server, and view its response times and
+overall calls.
+* The `virt-operator` pod cannot reach the API server. This is commonly caused
+by DNS issues on the node and networking connectivity issues.
+
+[discrete]
+[id="impact-virtoperatorresterrorsburst_{context}"]
+== Impact
+
+Cluster-level actions, such as upgrading and controller reconciliation, might
+not be available.
+
+However, customer workloads, such as virtual machines (VMs) and VM instances
```

Comment:
Both seem like they could be correct, but just checking that this is intentionally "customer workloads" and not supposed to be "custom workloads". I see 1 instance of "custom workloads" in this PR and 2 of "customer workloads"

Reply:
customer workloads is correct. The original writers wrote "custom workloads" by mistake. Thanks for catching one we missed!

Reply:
To be honest, I don't like "customer workloads" either. "Workloads" would be better. Will update.